client/state/ui/guided-tours/test/config.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Internal dependencies
+ */
+import { isNewUser } from 'state/ui/guided-tours/selectors';
+import { getSectionName } from 'state/ui/selectors';
+
+const tours = {
+	main: {
+		meta: {
+			version: 'test',
+			path: '/',
+			context: state => isNewUser( state ),
```

Comment:
That's pretty interesting. I seem to remember circular dependencies resolving to `null`. I think we can just deal with the wrapping now :)